Objective-C/Internal/CBLDatabase+Internal.h

```diff
@@ -17,6 +17,8 @@
 #import "CBLMutableDocument.h"
 #import "CBLDocumentChange.h"
 #import "CBLJSONCoding.h"
+#import "CBLReplicator.h"
```

Comment:
You don't need to import these headers; just use `@class` to declare the class name.

Reply:
I couldn't make this a forward declaration because I need to call "stop" on the replicator objects

Reply:
Not in this header, though! The header just needs to know that `CBLReplicator` is a class name. In the .m file(s) you would import the headers.